sources/packages/api/src/services/sfas/sfas-part-time-application.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Inject, Injectable } from "@nestjs/common";
+import { Connection } from "typeorm";
+import { DataModelService } from "../../database/data.model.service";
+import { SFASPartTimeApplications } from "../../database/entities";
+import { InjectLogger } from "../../common";
+import { LoggerService } from "../../logger/logger.service";
+import { getUTC } from "../../utilities";
+import { SFASDataImporter } from "./sfas-data-importer";
+import { SFASRecordIdentification } from "../../sfas-integration/sfas-files/sfas-record-identification";
+import { SFASPartTimeApplicationRecord } from "../../sfas-integration/sfas-files/sfas-part-time-application-record";
+
+/**
+ * Manages the part time application data related to an individual/student in SFAS.
+ */
+@Injectable()
+export class SFASPartTimeApplicationsService
+  extends DataModelService<SFASPartTimeApplications>
+  implements SFASDataImporter
+{
+  constructor(@Inject("Connection") connection: Connection) {
+    super(connection.getRepository(SFASPartTimeApplications));
+  }
+
+  /**
+   * Import a record from SFAS. This method will be invoked by SFAS
+   * import processing service when the record type is detected as
+   * RecordTypeCodes.ApplicationDataRecord.
```

Comment:
I believe that this should be updated to RecordTypeCodes.PartTimeApplicationDataRecord, right?